types/service/configuration.go

```diff
@@ -91,12 +92,50 @@ type ConnectorServiceConfiguration struct {
 	EndToEndEncryptionEnabled      bool          `json:"end_to_end_encryption_enabled"`
 	RecordingEnabled               bool          `json:"recording_enabled"`
 	Upstream                       Configuration `json:"upstream_configuration"`
+	PrivateNetworkIPv4             *string       `json:"private_network_ipv4"`
```

Comment:
I think using strings instea of ptr would be easier to manage, You can also add omitempty tags here.